src/main/java/com/rocketden/main/dto/room/RoomDto.java

```diff
@@ -19,4 +19,5 @@
     private List<UserDto> inactiveUsers;
     private boolean active;
     private ProblemDifficulty difficulty;
+    private long duration;
```

Comment:
nit: Use `Long` instead of `long` here, _or_ replace the fields elsewhere to use `long`?